notification/.prettierignore

```diff
@@ -5,3 +5,4 @@ package.json
 package-lock.json
 .prettierignore
 .eslintrc
+.nyc_output
```

Comment:
Same here, We should delete this file.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1000/commits/b0b1d5cb42203115281398c7718a5ab3e03accc4